markovflow/sde/sde_utils.py

```diff
@@ -133,3 +132,65 @@ def linearize_sde(
         state_offsets=state_offsets,
         chol_process_covariances=chol_process_covariances,
     )
+
+
+def KL_sde(sde_p: SDE, A_q: TensorType, b_q: TensorType, m: TensorType, S: TensorType, dt: float,
```

Comment:
Thinking about names again
KL_mgp_to_sde?
in practice it is KL(q|p) with q a markovian gp
KL_sde is a bit too generic

Reply:
maybe the arguments could be m_q, S_q?
Is there a reasons why you don't pass a ssm?
Then you could extract A_q, b_q and compute m, S from it.

Reply:
I agree with the name of the function being generic so will change it. :+1: 

In terms of why not passing the SSM. Basically, the A and b parameters and not always of the `q` SSM. If we think about the linear loss term as well in the CVI-SDE class where this function is used currently, we need to calculate 

`E_{q}[f_p - f_l]^2_{Sigma^{-1}}` 

so A, b are for the linear SDE and m, S are of the posterior q.

Reply:
I see, the docstrings is a bit confusing so will modify it too.

Reply:
The function can be used for KL[q||p] but it is more generic in a way how we use it to calculate the KL_linearization_loss term.

Reply:
Modified the doc string and the function name for it to be more clear.

Reply:
`squared_drift_difference_along_Gaussian_path` ? 
How about you pass `m,S` into a Gaussian object (broadcasting)
And you create a Similar wrapper class for a `LinearDrift` where you pass the A, b (broadcasting)

Reply:
This is really just about not having too many variables in the arguments list